internal/api/handlers/count.go

```diff
@@ -0,0 +1,42 @@
+package handlers
+
+import (
+	"awans.org/aft/internal/api/parsers"
+	"awans.org/aft/internal/bus"
+	"awans.org/aft/internal/db"
+	"awans.org/aft/internal/server/lib"
+	"github.com/json-iterator/go"
+	"net/http"
+)
+
+type CountHandler struct {
+	db  db.DB
+	bus *bus.EventBus
+}
+
+func (s CountHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) (err error) {
+	modelName, foBody, err := unpackArgs(r)
+	if err != nil {
+		return err
+	}
+
+	tx := s.db.NewRWTx()
+	p := parsers.Parser{Tx: tx}
+
+	op, err := p.ParseCount(modelName, foBody)
+	if err != nil {
+		return
+	}
+
+	s.bus.Publish(lib.ParseRequest{Request: op})
+
+	out, err := op.Apply(tx)
+	if err != nil {
+		return
+	}
+
+	bytes, _ := jsoniter.Marshal(&SummaryResponse{BatchPayload{Count: out}})
```

Comment:
Does this return
```{ "BatchPayload": {"count": 2}}```

or just 
```{"count": 2}```

(I'd like to see the latter, I think)

Reply:
Same feedback on each of the others..

Reply:
it returns the former per prima spec

Reply:
I think that's a misreading of the prisma spec -- BatchPayload is just a type, not a returned key (see for example the other types in the docs like ```export type User````)